ibft/proto/params.go

```diff
@@ -22,7 +22,7 @@ func (keys PubKeys) Aggregate() bls.PublicKey {
 //DefaultConsensusParams returns the default round change duration time
 func DefaultConsensusParams() *ConsensusParams {
 	return &ConsensusParams{
-		RoundChangeDuration: int64(time.Second * 5),
+		RoundChangeDuration: int64(time.Second * 3),
```

Comment:
not risky? I'm not sure about time limits

Reply:
let's leave it for now as is